### PR TITLE
Quick fixes to the gradio/lite guide

### PR DESCRIPTION
**Pull request by abidlabs**

### CODE REVIEW

File: .changeset/lite-fake-patch.md
Comment (abidlabs) [inline .changeset/lite-fake-patch.md:1]: So that we can trigger a patch release easily and bring the Gradio/lite Guide to gradio.app/guides

**Comment by aliabd** [comment]
@abidlabs i linked to the old name, let me fix here

**Comment by abidlabs** [comment]
Oh right, my bad. Thanks

File: guides/08_other-tutorials/gradio-lite.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Gradio-Lite: Serverless Gradio Running Entirely in Your Browser
 
+Tags: SERVERLESS, BROWSER, PYIODIDE
```
Comment (whitphx) [inline guides/08_other-tutorials/gradio-lite.md:3]: Plz replace "PYIODIDE" to "PYODIDE" 👍

**Review by aliabd — APPROVED** [review]
lgtm

**Review by yvrjsharma — APPROVED** [review]
LGTM!

**Pull request merged**